README.md

```diff
@@ -28,22 +60,93 @@ This initializes your credentials for use with both the CLI and the
 Python client.
 
 After doing this, in Python you can simply instantiate the Promethium Client
-directly using:
+directly in python using:
```

Comment:
Nit: Small casing typo for Python here (and a few more below).

Reply:
Done